resources/views/emails/spacemanagermorningreport.blade.php

```diff
@@ -0,0 +1,23 @@
+@component('mail::message')
+# Good Morning!
+
+There @choice('is|are', count($space->activeVisits)) currently {{ count($space->activeVisits) }} active @choice('visit|visits', count($space->activeVisits)) for the {{ $space->name }} space.
+
+@component('mail::panel')
+## Active Visits for {{ $space->name }}
+@if(count($space->activeVisits) > 0)
+@foreach($space->activeVisits as $visit)
+- **{{ $visit->user->first_name }} {{ $visit->user->last_name }}** (since {{ $visit->in_time }})
```

Comment:
It would be nice to say how long ago the visit started ("8 hours ago") if that's easy to do.

Reply:
Should we exclude visits that started "recently", like within the previous 8 hours or so?